calibrations/mbd/prorun.sh

```
@@ -0,0 +1,206 @@
+#!/usr/bin/bash
+
+#inputs=dst_triggered_event_run2auau-00054920.list, for example
+
+if [[ $# -lt 1 ]]
+then
+  echo Error, need to specify input file
+  exit -1
```

Shellcheck (0.11.0)

[error] Can only exit with status 0-255. Other data should be written to stdout/stderr.

(SC2242)